spacelift/resource_scheduled_task.go

```diff
@@ -0,0 +1,270 @@
+package spacelift
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/pkg/errors"
+	"github.com/shurcooL/graphql"
+
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal"
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal/structs"
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal/validations"
+)
+
+type ScheduledTask struct {
+	At       *int
+	Command  string
+	Every    []string
+	Timezone string
+}
+
+func resourceScheduledTask() *schema.Resource {
+	return &schema.Resource{
+		Description: "" +
+			"`spacelift_scheduled_task` represents a scheduling configuration " +
+			"for a Stack. It will trigger task on the given schedule or timestamp",
+
+		CreateContext: resourceScheduledTaskCreate,
+		ReadContext:   resourceScheduledTaskRead,
+		UpdateContext: resourceScheduledTaskUpdate,
+		DeleteContext: resourceScheduledTaskDelete,
+
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"stack_id": {
```

Comment:
Same here - maybe we should include the `schedule_id` as a computed attribute?

Reply:
addressed